packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -509,16 +509,18 @@ class _TreemapState extends State<Treemap> {
     }
   }
 
-  Text buildNameAndSizeText({
+  RichText buildNameAndSizeText({
```

Comment:
Can we return a `TextSpan` instead of `RichText` (that way we don't have to wrap our `TextSpan` in `RichText`)?

Reply:
RichText is being used to support buildCell and buildTitleText request for a widget.